modules/persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure_{context}"]
+= Pre-provisioning across subscriptions for Azure File by creating a PV and PVC:
+
+.Prerequisites
+* Installed Azure cluster with the service principal or managed identity as an Azure identity
```

Comment:
Do we mean OCP cluster on Azure? Is it standard policy to call it "Azure Cluster"

Reply:
I think you're right. I've changed it.